pkg/webui/console/components/events/previews/shared/json-payload/index.js

```diff
@@ -84,6 +84,10 @@ KeyValue.defaultProps = {
 const JSONPayload = React.memo(props => {
   const { data } = props
 
+  if (!data) {
```

Comment:
`data` is set to be required in `prop-types`. imo, we should whether event has an entry to for the `JSONPayload` component before rendering it.

Reply:
We do that, this is more meant as a fallback, to prevent the whole event from crashing.